site/public/index.html

```diff
@@ -343,13 +341,22 @@ <h2>Information for education providers</h2>
 					data-component="rc-most-viewed-carousel"
 					data-props='{"slides":[{"title":"Keeping our schools open","description":"Minister Tehan’s message for parents, students, teachers and school leaders.","linkUrl":"https://www.dese.gov.au/news/coronavirus-covid-19"},
```

Comment:
Should "Keeping our schools open" tile also be reformatted?

Reply:
Yes - will do now.